### PR TITLE
Implemented the new method for my account.

### PR DESCRIPTION
**Pull request by ionutvmi**

### CODE REVIEW

File: src/main/java/ro/tefacprogramator/m4/exampleapp/controllers/MyAccountController.java [hunk 1]
```diff
@@ -5,5 +5,7 @@
  */
 public class MyAccountController {
 
-    
+    public void showSomething() {
+        System.out.println("Test message");
```
Comment (ionutvmi) [inline src/main/java/ro/tefacprogramator/m4/exampleapp/controllers/MyAccountController.java:9]: This is a nice new message.

File: src/main/java/ro/tefacprogramator/m4/exampleapp/controllers/MyAccountController.java [hunk 2]
```diff
@@ -5,5 +5,7 @@
...
+    }
```
Comment (standiaa) [inline src/main/java/ro/tefacprogramator/m4/exampleapp/controllers/MyAccountController.java:10]: message :)

File: src/main/java/ro/tefacprogramator/m4/exampleapp/controllers/MyAccountController.java [hunk 3]
```diff
@@ -5,5 +5,7 @@
  */
 public class MyAccountController {
```
Comment (IordanDenis) [inline src/main/java/ro/tefacprogramator/m4/exampleapp/controllers/MyAccountController.java:7]: Test comment

File: src/main/java/ro/tefacprogramator/m4/exampleapp/controllers/MyAccountController.java (hunk 2 above)
Comment (ionutvmi) [reply src/main/java/ro/tefacprogramator/m4/exampleapp/controllers/MyAccountController.java:10]: nice

**Comment by ionutvmi** [comment]
Hello,

This is a longer message,

I can put some code

```
    public void showSomething() {
        System.out.println("Test message");
    }
```

File: src/main/java/ro/tefacprogramator/m4/exampleapp/controllers/MyAccountController.java [hunk 4]
```diff
@@ -5,5 +5,7 @@
...
 
-    
+    public void showSomething() {
```
Comment (fabiancostin) [inline src/main/java/ro/tefacprogramator/m4/exampleapp/controllers/MyAccountController.java:8]: test

File: src/main/java/ro/tefacprogramator/m4/exampleapp/controllers/MyAccountController.java (hunk 1 above)
Comment (NicuNedelea) [reply src/main/java/ro/tefacprogramator/m4/exampleapp/controllers/MyAccountController.java:9]: Nice !!!

**Comment by claudiastoica** [comment]
Test message :)

**Review by IordanDenis — APPROVED** [review]
Approved - test

**Review by fabiancostin — APPROVED** [review]

**Review by trustedbrand — APPROVED** [review]
Approve test

**Review by standiaa — APPROVED** [review]
all good

**Review by NicuNedelea — APPROVED** [review]

**Review by claudiastoica — APPROVED** [review]
Test

**Pull request merged**